index.js

```diff
@@ -8,7 +8,13 @@ const MongoClient = MongoDb.MongoClient
 const ObjectId = MongoDb.ObjectId
 
 function decorateFastifyInstance (fastify, client, options, next) {
-  fastify.addHook('onClose', (fastify, done) => client.close(done))
+  fastify.addHook('onClose', (fastify, done) => {
```

Comment:
We can skip the hook registration at all.

Reply:
👍 , I'll skip the hook then.